tendermint/src/lite/verifier.rs

```diff
@@ -377,12 +380,13 @@ mod tests {
         next_vals_vec: Vec<usize>,
         height: u64,
     ) -> MockState {
-        let time = init_time();
+        // time has to be increasing:
+        let time = init_time() + Duration::new(height * 2, 0);
         let vals = &MockValSet::new(vals_vec.clone());
         let next_vals = &MockValSet::new(next_vals_vec);
         let header = MockHeader::new(height, time, vals.hash(), next_vals.hash());
         let commit = MockCommit::new(header.hash(), vals_vec);
```

Comment:
Similarly to the single cases we actually want to able to provide the commit vals separately from the other vals.

Reply:
Done by @Shivani912 in #152